### PR TITLE
Us 33 05 add update

### PR DESCRIPTION
**Pull request by Loic-Code**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/View/Admin/Includes/_show_stages.html.twig [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                <td>{{ stage.age }}</td>
+                <td>
+                    <a class="btn btn-sm btn-secondary mb-1 col-8" href="/AdminStage/edit/{{ stage.id }}">Editer</a>
+                        <form action="/adminStage/delete/{{stage.id}}" method="POST" onSubmit="return confirm('Voulez-vous supprimer ?');">
+                        <button class="btn btn-sm btn-danger mb-1 col-8">Supprimer</button>
+                    </form>
+                </td>
+            </tr>
+        {% endfor %}
+    </tbody>
```
Comment (LucyMerlier) [inline src/View/Admin/Includes/_show_stages.html.twig]: indentation

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by bouboumael — DISMISSED** [review]

File: src/View/Admin/Includes/_show_stages.html.twig [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<div class="container mt-5 pt-5">
+    <table class="table table-striped table-hover text-center align-middle">
+    <thead>
```
Comment (LucyMerlier) [inline src/View/Admin/Includes/_show_stages.html.twig]: indentation inside table tag

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by bouboumael — DISMISSED** [review]

**Review by ChristianOlmedo — DISMISSED** [review]

**Review by LucyMerlier — DISMISSED** [review]

**Pull request merged**